lib/core/src/lib/form/components/widgets/core/form.model.ts

```diff
@@ -387,9 +387,11 @@ export class FormModel implements ProcessFormModel {
     }
 
     private handleSingleField(field: FormFieldModel, formFieldModel: FormFieldModel[]): void {
-        formFieldModel.push(field);
-        if (field.fields) {
-            this.processFields(Object.values(field.fields), formFieldModel);
+        if (field instanceof FormFieldModel) {
```

Comment:
i would also suggest to move this condition to separate method, like 
    private isFormField(field: ContainerModel | FormFieldModel): field is FormFieldModel {
        return field instanceof FormFieldModel;
    }
    
    and use it at the else if in the processFields method, to be consistent with the use of isContainerField.